spec/models/ext_management_system_spec.rb

```diff
@@ -439,22 +440,17 @@
 
         deliver_queue_message
 
-        expect(MiqQueue.count).to eq(1)
-        expect(MiqQueue.last.deliver_on).to_not be_nil
+        expect(MiqQueue.count).to eq(0)
```

Comment:
I think you need to update the description of this test since the functionality has changed.